platform/android/java/lib/src/org/godotengine/godot/utils/PermissionsUtil.java

```diff
@@ -0,0 +1,159 @@
+package org.godotengine.godot.utils;
+
+import android.Manifest;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.content.pm.PermissionInfo;
+import android.os.Build;
+import android.support.v4.content.ContextCompat;
+import java.util.ArrayList;
+import java.util.List;
+import org.godotengine.godot.Godot;
+
+/**
+ * This class includes utility functions for Android permissions related operations. You should request permissions
+ * which has defined in manifest. With this util you can request permission which protection level is
+ * dangerous.
```

Comment:
You can remove the following two sentences since the methods' descriptions already cover them:
```
You should request permissions which has defined in manifest. With this util you can request permission which protection level is dangerous.
```

Reply:
Fixed.